task/10-katas-1-tasks.js

```diff
@@ -113,7 +227,20 @@ function getZigZagMatrix(n) {
  *
  */
 function canDominoesMakeRow(dominoes) {
-    throw new Error('Not implemented');
+    let maxVal=dominoes.reduce((acc, val)=> Math.max(Math.max(val[1], val[0]), acc),0);
+    let matrixOfNe = Array(maxVal+1).fill(0).map((x)=>Array(maxVal+1).fill(0));
+    
+    for (let el of dominoes){
+        matrixOfNe[el[0]][el[1]]++;
+        matrixOfNe[el[1]][el[0]]++;
+    }
+    for (let i=0; i<matrixOfNe.length; i++){
+        if (matrixOfNe[i][i]!=0 && matrixOfNe[i][i] == matrixOfNe[i].reduce((acc, val)=>acc+val,0)) return false;
```

Comment:
"operator !== is always preferable
sometimes ''!=' may cause an error
(check at console :
'0' != 0
and '0' !== 0)"